<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Port</class>
 <widget class="QMainWindow" name="Port">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Port</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="ledit_send"/>
           </item>
           <item>
            <widget class="QPushButton" name="btn_send">
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>89</width>
         <height>23</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>COM1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM6</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>89</width>
         <height>23</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1200</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="btn_clear">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>270</y>
         <width>89</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_open">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>160</y>
         <width>89</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_close">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>200</y>
         <width>89</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_status">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>271</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Status:Disconnected</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
